src/main/java/com/littleBeasts/entities/LitiPlayer.java

```diff
@@ -50,6 +51,17 @@ public static LitiPlayer instance() {
         return litiPlayerInstance;
     }
 
+    public void addItemToPlayerInventory() throws NoPlaceInInventoryException {
```

Comment:
I don't really get the purpose of this method because it always adds the same items to the inventory. Is this a method for testing purposes?

Reply:
is removed